lib/dependency-manager-adapters/npm.js

```diff
@@ -43,7 +67,7 @@ module.exports = CoreObject.extend({
           name: dep,
           versionExpected: deps[dep],
           versionSeen: adapter._findCurrentVersionOf(dep),
-          packageManager: 'npm'
+          packageManager: adapter.configKey
```

Comment:
I think that this `packageManager` key is used just for displaying versions of packages being installed in the results table which is drawn to the user after all test routines are finished. The fact that the displayed table has proper data is untested btw.